content/en/news/security/istio-security-2023-004/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: ISTIO-SECURITY-2023-004
+subtitle: Security Bulletin
+description: CVE reported by Envoy.
+cves: [CVE-2023-48887]
```

Comment:
```suggestion
cves: [CVE-2023-44487]
```

Reply:
The CVE is linked to 44487 but is texted as 48887. I believe 44487 is correct but we're mixing between the two